builds/e2e/e2e.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
 
     pool:
       name: $(pool.custom.name)
-      demands: rpi3-e2e-tests-1.1
+      demands: rpi3-e2e-tests
```

Comment:
Since the 1.1 dedicated pi became unresponsive to ssh, I synced with Bear and we are falling back to the default pool. The most recent test run passed on arm. CentOS is not related to this PR, but is an unrelated issue.
https://dev.azure.com/msazure/One/_build/results?buildId=48735415&view=results

Reply:
Do we know why it became un-responsive ? My reasoning for switching to a dedicated 1.1 agent was 2 fold
1. Prevent any failures due to improper cleanup when going from 1.1 Test to 1.2/master or vice-versa
2.  I had noticed intermittent failures on default pool (1 in every 3 or 4 runs) 

Not planning to hold this PR, but I think we should investigate what went wrong there

Reply:
Usually when the pis become unresponsive it is either disk space or a resource consumption problem that might go away later. I discussed with Bear and we thought 1.1 and 1.2 branch E2E test frameworks can handle uninstalling / cleaning up resources from same/different branches. Is there a reason you may think differently?

Also, do you remember why there were intermittent failures in the default pool?

Reply:
And yeah we can followup on this agent. I will wait to see if it becomes responsive later today. If not, we will need to go into the office and restore it. In this case the work will be done as part of the pi stabilzation PBI.

Reply:
It was the workload socket issue that hughes recently fixed in 1.1 ( When upgrading to 1.2) . Now that I think about it, since that issue has been fixed, we can go back to using the default pool.